Formula/a/audacious.rb

```diff
@@ -54,14 +54,18 @@ class Audacious < Formula
   depends_on "ffmpeg"
   depends_on "flac"
   depends_on "fluid-synth"
+  depends_on "gdk-pixbuf"
+  depends_on "gettext"
```

Comment:
Runtime `gettext` is usually macOS-only given most linkage is to `libintl`, which is part of system `glibc` on Linux.